Assets/MouseLook.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MouseLook : MonoBehaviour
+{
+    public float mouseSensitivity = 100f;
```

Comment:
```suggestion
    public const float MouseSensitivity = 100f;
```

public properties are CamelCased